examples/spark/examples_utils.py

```diff
@@ -120,11 +129,11 @@ def get_spark_session(partitions_num: Optional[int] = None):
         spark_sess = (
             SparkSession
             .builder
-            .master("local[4]")
+            .master(f"local[{partitions_num}]")
             # .config("spark.jars.packages",
-            #         "com.microsoft.azure:synapseml_2.12:0.9.5,io.github.fonhorst:spark-lightautoml_2.12:0.1")
+            #         "com.microsoft.azure:synapseml_2.12:0.9.5,io.github.fonhorst:spark-lightautoml_2.12:0.1.1")
```

Comment:
please remove unused comments

Reply:
moved to an example comment slightly above